HCCGo/app/js/ConnectionService.js

```diff
@@ -872,14 +872,24 @@ connectionModule.factory('connectionService',['$log', '$q', '$routeParams', '$lo
                 }
        });
 
-   }
+   };
 
+   var quickDownload = function(remotePath, localPath) {
```

Comment:
Also, could you add jsdoc comments to this function, now that we have auto compilation.